io.github.mrmyhuang.cbetar2.metainfo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file should be also submitted upstream to be managed there. It's not flathub specific

Reply:
fix